src/opnsense/service/templates/OPNsense/IDS/suricata.yaml

```diff
@@ -942,7 +895,7 @@ app-layer:
       #encryption-handling: default
 
     pgsql:
-      enabled: no
+      enabled: {{ 'yes' if 'pgsql' in eveLog_types_enabled else 'no' }}
```

Comment:
shouldn't this also move into the loop? I expect it duplicates the item now, or am I mistaken?

Reply:
@0xThiebaut ok, thanks, I'll need to try this on my end before merging. these yaml files can be sensitive in my experience.

Reply:
Thanks, I really appreciate the time and care you're taking for community contributions!

Reply:
@0xThiebaut you're welcome, we can't pull-in everything, but do make an effort when possible.